backend/django/create-model-and-api.mo.md

```diff
@@ -0,0 +1,219 @@
+# [MO] Create a model and an api route (~20 mins)
+
+## Owner: Alice Breton
+
+## Prerequisites (~2 mins)
+
+- Have a django project installed, [see this](https://github.com/bamlab/dev-standards/blob/master/backend/django/getting-started.mo.md).
+- Have an app (in this example Publications).
+- In this example we will use an already existing model 'city' in a 'locations' app, to show how to integrate other models with foreign keys.
+
+## Steps  
+
+- In this tutorial we will create a News model, add it to the Django admin back-office and an API route to get all the pieces of news that have been created.
+- To give a full example, we will ==== REMPLIR
+
+### Create a News model *(~10 min)*
+
+- In this example we will create a new model called News, with different attributes as you will see bellow. In the models.py file of your app (in our case Publications) create a new model. 
+
+ourproject/publications/models.py
+```python
+from django.db import models
+
+class News(models.Model):
+    title = models.CharField(max_length=50)   #CharField is a single line form in the admin; max_length speaks for itself
+    description = models.TextField(max_length=1000)   #TextField is a scrollable paragraph in the admin
+    created_at = models.DateTimeField(auto_now_add=True)    #auto_now_add generates automatically a timestamp when you create your create the news
+    updated_at = models.DateTimeField(auto_now=True)    #auto_now updates the timestamp everytime you change the news
+    city = models.ForeignKey('locations.city')    #All the cities of the City model of the locations app
```

Comment:
"The many to one relationship: each news are linked to one city. Two news can have the same city."

Reply:
"The many to one relationship: every piece of news is linked to one city, two pieces of news can have the same city." ?